modules/ossm-deploying-multiple-control-planes.adoc

```diff
@@ -0,0 +1,152 @@
+// Module included in the following assemblies:
+// install/ossm-using-multiple-control-planes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploying-multiple-control-planes_{context}"]
+= Deploying multiple control planes
+:context: ossm-deploying-multiple-control-planes
+
+This procedure demonstrates how to configure a cluster to host two control planes in separate system namespaces. Each control plane will manage specific application workloads running in designated namespaces. The namespaces are associated with a control plane using revision and discovery selector configurations.
+
+.Prerequisites
+* The {SMProduct} operator is installed.
+* An {istio} CNI resource is created.
+* The `istioctl` binary is installed on your localhost.
+
+.Procedure
+
+. Create the namespace for the first {istio} control plane called `istio-system-1`:
++
+[source,terminal]
+----
+$ oc create namespace istio-system-1
+----
+
+. Add label to the namespace:
++
+[source,terminal]
+----
+$ oc label namespace istio-system-1 usergroup=usergroup-1
+----
+
+. Create a YAML file named `istio-1.yaml` with the name `mesh-1` and the discoverySelector `usergroup-1`:
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+kind: Istio
+apiVersion: sailoperator.io/v1alpha1
+metadata:
+  name: mesh-1
+spec:
+  namespace: istio-system-1
+  values:
+    meshConfig:
+      discoverySelectors:
+        - matchLabels:
+            usergroup: usergroup-1
+----
+
+. Create `Istio` resource:
++
+[source,terminal]
+----
+$ oc apply -f istio-1.yaml
+----
+
+. Create a namespace for the second Istio control plane called `istio-system-2`:
++
+[source,terminal]
+----
+$ oc create namespace istio-system-2
+----
+
+. Add label to the second namespace:
++
+[source,terminal]
+----
+$ oc label namespace istio-system-2 usergroup=usergroup-2
+----
+
+. Create a YAML file named `istio-2.yaml`:
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+kind: Istio
+apiVersion: sailoperator.io/v1alpha1
+metadata:
+  name: mesh-2
+spec:
+  namespace: istio-system-2
+  values:
+    meshConfig:
+      discoverySelectors:
+        - matchLabels:
+            usergroup: usergroup-2
+----
+
+. . Create `Istio` resource:
++
+[source,terminal]
+----
+$ oc apply -f istio-2.yaml
+----
+
+. Deploy a policy for workloads in the `istio-system-1` namespace to only accept mutual TLS traffic `peer-auth-1.yaml`:
```

Comment:
```suggestion
. While we now have two independently managed service meshes in the same cluster, by default, workloads are able to communicate freely between meshes with unencrypted traffic, which may not be desirable. To lock down the `mesh-1` data plane such that only mesh-internal mutual TLS traffic is accepted,  deploy a `PeerAuthentication` resource to require mTLS traffic for workloads in the `istio-system-1`. In this case, called `peer-auth-1.yaml`:
``` 
@FilipB Can you confirm that I have the above accurate? I felt like more explanation was needed as to what this policy is doing, because we don't talk about PeerAuthentications elsewhere. It's not that clear in the upstream doc either, because it assumes you already know about PeerAuthentications, which i don't want to assume the reader knows about here.